File: js/practice-01.js
// 配列を用意
const list1 = Array.from(Array(37).keys()).slice(1);
const list2 = Array.from(Array(26).keys()).slice(1);

const result = [];
for (let e1 of list1) {
    for (let e2 of list2) {
        // 10 で割った時の「余り」（剰余）を比較
        if (e1 % 10 == e2 % 10) {
            console.log(`[match] (${e1}, ${e2})`);
            result.push([e1, e2]);
        }
    }
}

console.log(`===================================`);
console.log(`[一の位が一致する組]: ${result.length}`);

File: python/practice_01.py
# 配列データを用意
list1 = list(range(1, 37))
list2 = list(range(1, 26))

result = []
for e1 in list1:
    for e2 in list2:
        # 10 で割った時の「余り」（剰余）を比較
        if e1 % 10 == e2 % 10:
            print(f"[match] ({e1}, {e2})")
            result.append((e1, e2))

print(f"[一の位が一致する組]: {len(result)}")
